pages/index.tsx

```diff
@@ -47,18 +51,15 @@ export default function Home() {
 
```

Comment:
Maybe it's better to wrap it in a div and use max width for large screens? It looks rather wide here:

![image](https://user-images.githubusercontent.com/103034384/237021184-05b86e84-ad7b-4c24-9b13-b278af0990c2.png)